pkg/apiclient/rollout/rollout.proto

```diff
@@ -148,8 +148,16 @@ message JobInfo {
   k8s.io.apimachinery.pkg.apis.meta.v1.Time startedAt = 5;
 }
 
+message SpecAndStatus {
```

Comment:
This type should be `AnalysisRunSpecAndStatus` but the name should be `specAndStatus` in the response